### ISSUE TITLE
Getting Error logs with very few details.

### ISSUE THREAD

**Issue by sbnajardhane** [rwynn/monstache#263]
```
ERROR 2019/07/17 02:01:39 not found
ERROR 2019/07/17 02:03:39 not found
ERROR 2019/07/17 09:59:49 not found
ERROR 2019/07/17 10:01:49 not found
ERROR 2019/07/17 10:03:49 not found
ERROR 2019/07/17 17:05:54 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:07:26 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:07:38 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:02 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:15 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:22 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:27 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:29 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:31 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:33 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:34 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:37 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:38 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:38 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:39 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:39 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:41 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:42 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:42 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:43 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:44 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:44 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:45 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:46 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:47 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:49 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:49 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:50 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:51 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:52 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:53 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:13:58 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:19:49 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:19:57 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/17 17:55:59 not found
ERROR 2019/07/17 17:57:59 not found
ERROR 2019/07/17 17:59:59 not found
ERROR 2019/07/18 00:40:09 not found
ERROR 2019/07/18 00:41:14 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/18 00:42:09 not found
ERROR 2019/07/18 00:44:09 not found
ERROR 2019/07/18 08:24:19 not found
ERROR 2019/07/18 08:26:19 not found
ERROR 2019/07/18 08:28:19 not found
ERROR 2019/07/18 16:26:29 not found
ERROR 2019/07/18 16:28:29 not found
ERROR 2019/07/18 16:30:29 not found
ERROR 2019/07/18 16:32:29 not found
ERROR 2019/07/18 16:34:29 not found
ERROR 2019/07/18 16:36:29 not found
ERROR 2019/07/19 00:10:39 not found
ERROR 2019/07/19 06:41:30 elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]
ERROR 2019/07/19 06:43:49 not found
```

I am getting such error logs continuously. Unable to figure out the issue. 
Could you please help me to provide direction for debugging?
Docker version: 4.11.3

**Comment by jodevsa** [comment]
Automatic index creation might be disabled 
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#index-creation

**Comment by sbnajardhane** [comment]
I am not creating any indexes on the fly.
I am listening to the only one collection from my MongoDB and I have already created an index for the same in Elasticsearch.
```namespace-regex = 'test_db.users'```
Also, I have observed that few (10-15) records failed to update in Elasticsearch when I perform any bulk update operation on my users' collection (around 100k users). That's why I am looking into the logs. This has happened couple of time.
